tests/python/pants_test/option/test_global_options.py

```diff
@@ -0,0 +1,22 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.option.global_options import GlobMatchErrorBehavior
+from pants.util.objects import TypeCheckError
+from pants_test.base_test import BaseTest
+
+
+class GlobalOptionsTest(BaseTest):
+
+  def test_exception_glob_match_constructor(self):
```

Comment:
Ditto testing options parsing.

Reply:
I have removed this file, there is ample testing already for this functionality.